packages/nimbus/src/components/radio-input/radio-input.stories.tsx

```diff
@@ -33,10 +33,10 @@ export const Base: Story = {
   render: () => (
     <Stack gap="1000">
       <RadioInput.Root
-        name="storybook-radio-base"
+        aria-label="storybook-radio-base"
```

Comment:
adding aria-labels in these tests to get rid of the `react-aria` 'you must label' warnings in the tests.